ci/tasks/check-if-usn-is-in-github.yml

```
@@ -0,0 +1,16 @@
+  - name: bosh-stemcells-ci
+  - name: usn
+  - name: usn-gh-json
+  - name: unfound-usns
+
+outputs:
+  - name: found-usns
+  - name: updated-unfound-usns
+
+run:
+  path: bosh-stemcells-ci/ci/tasks/check-if-usn-is-in-github.sh
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)